docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md

```diff
@@ -36,7 +36,13 @@ ms.author: mikeray
   
  Only members of the [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] sysadmin role can connect using the DAC.  
   
- The DAC is available and supported through the **sqlcmd** command-prompt utility using a special administrator switch (**-A**). For more information about using **sqlcmd**, see [Use sqlcmd with Scripting Variables](../../relational-databases/scripting/sqlcmd-use-with-scripting-variables.md). You can also connect prefixing **admin:** to the instance name in the format **sqlcmd -S admin:<*instance_name*>**. You can also initiate a DAC from a [!INCLUDE[ssManStudioFull](../../includes/ssmanstudiofull-md.md)] Query Editor by connecting to **admin:\<*instance_name*>**.  
+ The DAC is available and supported through the **sqlcmd** command-prompt utility using a special administrator switch (**-A**). For more information about using **sqlcmd**, see [Use sqlcmd with Scripting Variables](../../relational-databases/scripting/sqlcmd-use-with-scripting-variables.md). You can also connect prefixing **admin:** to the instance name in the format **sqlcmd -S admin:<*instance_name*>**. You can also initiate a DAC from a [!INCLUDE[ssManStudioFull](../../includes/ssmanstudiofull-md.md)] Query Editor by connecting to **admin:\<*instance_name*>**.
+ 
+> [!Note]  
+> To establish a DAC from SQL Server Management Studio
+> - Disconnect all connections to the related database

```

Comment:
> - Disconnect all connections to the related SQL Server instance, including the Object Explorer and all open query windows.

Reply:
Apologies for the delay.  GitHub notifications got marked as spam in an Outlook.com account (how is that possible!?).
Anyway, I would happily accept the change if I could figure out how.  I even phoned a friend and we're stumped.

Reply:
No worries! I think you need to rewrite your change to match my review, I don't think there's a way to automatically accept it.

Reply:
I can't figure it out. I'm going to manually take in the changes @bluefooted recommended in our private repository.

@ChadSimmons thanks for the submission.